crates/bevy_render/src/texture/image_texture_conversion.rs

```diff
@@ -221,7 +229,7 @@ mod test {
         let mut initial = DynamicImage::new_rgba8(1, 1);
         initial.put_pixel(0, 0, Rgba::from([132, 3, 7, 200]));
 
-        let image = Image::from_dynamic(initial.clone(), true);
+        let image = Image::from_dynamic(initial.clone(), true, false);
```

Comment:
How is false being cast to RenderAssetPersistentAccess::Unload implicitly here?

Reply:
It's not, guess I missed some, and CI is currently broken 😅